src/lib/swapper/LifiSwapper/getTradeQuote/getTradeQuote.ts

```diff
@@ -105,7 +73,7 @@ export async function getTradeQuote(
     toTokenAddress: buyLifiToken.address,
     fromAddress: receiveAddress,
     toAddress: receiveAddress,
-    fromAmount: thresholdedAmountCryptoLifi,
+    fromAmount: fromAmountCryptoLifiPrecision.toString(),
```

Comment:
I think this guy should really be:

```suggestion
    fromAmount: fromAmountLifiCryptoPrecision.toString(),
```